app/uk/gov/hmrc/identityverificationquestions/config/AppConfig.scala

```diff
@@ -35,6 +35,12 @@ class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig)
 
   def allowedUserAgentList: Seq[String] = getStringList("allowedUserAgentList").getOrElse(Seq.empty[String])
 
+  def deniedUserAgentListForP60: Seq[String] = getStringList("deniedUserAgentListForPaye").getOrElse(Seq.empty[String])
+  def deniedUserAgentListForSA: Seq[String] = getStringList("deniedUserAgentListForSA").getOrElse(Seq.empty[String])
+  def deniedUserAgentListForPayslip: Seq[String] = getStringList("deniedUserAgentListForPayslip").getOrElse(Seq.empty[String])
+  def deniedUserAgentListForNtc: Seq[String] = getStringList("deniedUserAgentListForNtc").getOrElse(Seq.empty[String])
+  def deniedUserAgentListForPaye: Seq[String] = getStringList("deniedUserAgentListForPaye").getOrElse(Seq.empty[String])
```

Comment:
Can you rename this to payslip rather than paye to make it consistence with evidence option's names